clients/drcachesim/tracer/tracer.h

```diff
@@ -230,6 +231,8 @@ extern size_t redzone_size;
 extern size_t max_buf_size;
 extern size_t buf_hdr_slots_size;
 
+extern uint64 trace_for_instrs_curr_phase;
```

Comment:
This should be std::atomic as it is modified multiple times during the course of the run and every thread needs to pick up the new value.

Reply:
I removed it because we can infer it from the tracing_mode value.